routers/private/hook.go

```diff
@@ -113,6 +113,13 @@ func HookPreReceive(ctx *macaron.Context, opts private.HookOptions) {
 					})
 					return
 				}
+				if !protectBranch.MergeBlockedByRejectedReview(pr) {
```

Comment:
Shouldn't this if-check be checking for `true`?

Reply:
Test cases say that also 😅